python/src/puml/templates/requirements/requirements_diagram.puml.jinja2

```diff
@@ -1,21 +1,17 @@
-@startuml
+@startuml {{req_spec.title}} Requirements Diagram
 
 mainframe **req** [{{package | default('Package')}}] **{{title}}** [{{subtitle | default('Requirements')}}]
 
-{% for req in requirements %}
-object "<<{{req.type}}>>{% if req.title %}\n**{{req.title}}**{% endif %}" as {{req.name}} {
-    id = {{req.id}}
-    Text = {{req.shall}}
-    {% for attribute in req.attributes %}
+object "<<{{req_spec.type}}>>{% if req_spec.title %}\n**{{req_spec.title}}**{% endif %}" as {{req_spec.name}} {
```

Comment:
```suggestion
object "<<{{req_spec.type}}>> {% if req_spec.title %}**{{req_spec.title}}**{% endif %}" as {{req_spec.name}} {
```

Reply:
```suggestion
object "<<{{req_spec.type}}>> **{{req_spec.title}}**" as {{req_spec.name}} {
```

Since we are populating the title, that if check can go away now.